### ISSUE TITLE
Noob--Mist won't work for me--have 1100 ETH need to get off of Poliniex

### ISSUE THREAD

**Issue by klm23** [ethereum/mist#545]
Running Win 10, large big new computer 64bit.  Ethereum 7.2 When I install the app Windows firewall asks me unblock everything, which I do. 

Next it tries to sync but it just stays on the image in the link below

http://picload.org/image/rgaapdwg/capture.png

Stayed like that for a day and I assume I should be seeing percentages of completion but I don't see it.
I give up and try again, but there is nothing in the unzipped folder to use as an icon to start mist, so the only l thing I can do is reinstall it.  I am stuck in this loop. 

When I choose to skip the sync the app opens--but I assume useless because it didn't sync. When I close the app it updates to a somewhere but I'm not savy enough to find out where all this stuff is in my computer.

Sorry I can't be more specific and I'm probably asking things already covered--but a lot of what is covered is over my head--but I'm very worried leaving all that ETH on an exchange.

I did figure out how to use mywallet.com--I guess I could wait for a more noob friendly wallet to come out and park my stuff there--(I know to backup password json, address and code-but now I want to try to invest in DAO and the instruction there say it is best for a noob, if not pretty much necessary for most all to this via mist?

Thanks so much in advance for help and for the work you do!

**Comment by luclu** [comment]
Hi @klm23 did your setup used to run before? Hence could you already open the wallet, play with sending ether and then withdraw from the exchange?

Try the following:
1. redownload and extract
2. try again with deactivated firewall (and AV) for test purpose.

**Comment by klm23** [comment]
Thanks so much for your response.  I have installed now several times and
did what you said and it still gets stuck on

http://picload.org/image/rgallrla/capture.png
Same result when I turn off firewalls.  Even with firewalls off when app
launches it still tells me to let firewall drop for this app

I assume it should have gone through a lengthy thing like BTC wallet where
you see the percentage of completion of sync but it never did that.
I followed Ethereum sponsored tutorial on youtube.

I don't even know how to get everything geth/mist related of the computer
and start from scratch as I guess it is like TOR--you can't "uninstal" it
The only way I can start it is to reinstal as there is no icon to launch in
the zip folder.  Never had it before on my computer...I can send screen
shots of what it looks like open if you wish.  Tried to set up DAO to track
tokens but nothing.  I found what to put for every line of that setup
except what to put for the D with the curly thing instead of dollar sign,
but the it shows the DAO with nothing in it.
I thought I would just forget about mist and use myetherwallet.com  But now
I've got 50,000 DAO tokens and I understand I need Mist to follow the
tokens and projects.  Mayke there is another way to do that without Mist?
If not, I'm starting to get concerned.  Is there anyone I can maybe pay
with tokens or ETH  in advance who would then spend a little time on the
phone with me to trouble shoot this?
Cheers,

Ken

On Tue, May 3, 2016 at 3:25 PM, Luca Zeug <notifications@github.com> wrote:

> Hi @klm23 <https://github.com/klm23> did your setup used to run before?
> Hence could you already open the wallet, play with sending ether and then
> withdraw from the exchange?
>
> Try the following:
> 1. redownload and extract
> 2. try again with deactivated firewall (and AV) for test purpose.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ethereum/mist/issues/545#issuecomment-216638337>
>



-- 
Ken Mogul
12768 Aviano Dr
Naples, FL 34105-4931
kenmogul23@gmail.com
Home: 239-384-9358
Text only-no voice (678) 664-8523

**Comment by wowor** [comment]
I have a similar issue on windows 10 on a new laptop.  I downloaded the latest 7.2 Mist release and it only finds and alternates between 1 or 2 peers and it can never finish downloading blocks no matter how long I leave it open.  Blocks seem to load up extremely slow, (hours) and it never finishes.  Last time I tried I made it to 400,000 blocks and then it got stuck.  I'm thinking it might be because I can't find enough peers.  Firewall and AV is off, no difference.  I previously used an earlier version of Mist on a different laptop with windows 7 and it worked fine.
